### DISCUSSION TITLE
Support of Apple silicone M1 M2 M3...

### DISCUSSION THREAD

**Discussion by Bigblock** [jopohl/urh#1129] [Q&A]
Has anybody tested URH with an Apple silicone Mac?

My intel Mac has died and i need a new machine. So from a performance perspective, a M / Apple silicone Mac seems to be the right decision to me.

Still own an old PC, but it's way too slow for URH.

Has anybody successfully tested URH on a modern Mac?

Thanks in advance.

**Comment by GraylockInc** [comment]
Following*

**Comment by n8henrie** [comment]
At least as packaged in nixpkgs, it's running great on Apple Silicon.
